test/openst/gnosisSafeModules/delayedRecoveryModule/resetRecoveryOwner.js

```diff
@@ -0,0 +1,114 @@
+// 'use strict';
+
+const assert = require('assert');
+
+const { ContractInteract, Helpers } = require('@openst/openst.js');
+const shared = require('../../../shared');
+const Deployer = require('../../../../src/deployer');
+const OpenST = require('../../../../src/openst');
+
+describe('DelayedRecoveryModule', async () => {
+  it('resets recovery owner', async () => {
+    const chainConfig = shared.chainConfig;
+    const connection = shared.connection;
+    const web3 = connection.auxiliaryWeb3;
+
+    // Required but not validated arguments
+    const mockAddresses = {
```

Comment:
As a general thinking, I am not sure that for any JLP test we should have a mocked values. Even if we know that it's not going to be used, we should stick to already existed contracts. I am more thinking about Organization and UtilityToken contracts here. What do you think?

Reply:
@pgev, @gulshanvasnani had the same question / takes a similar view as you. I'm not familiar with what should and should not concern smoke tests, so I will defer to you.

My reasoning:
- my nascent understanding of smoke tests is that they are to make sure it minimally works, which, to my mind, means not making the test dependent on unrelated concerns (e.g., setup of organizations, BT, UBT, etc.)
  - of course, perhaps this really turns on the definition of "it"—I was thinking "it" is resetting recovery owner, but if "it" is testing _practically_ resetting owner (e.g., in a properly-setup token economy), that's a different matter and then I agree with your position
- we wanted to move quickly with these tests, and incorporating those other concerns risked going slower

Reply:
To my understanding, smoke tests in JLP should work with "real" contracts, the ones we will deploy for production later on. I also see Banks point that we should move faster with those tests (I would rather want to see this test as a unit/integration test in openst.js). I would propose to merge this PR now with mock contracts and later revisit them.